PMurphy1940/ContractThis
# Login registration

# Description
Users may register or login. A user may register as a subcontractor.
## Type of change


# Testing Instructions
Log in with a known email and password, or create a new account using register. On register, you may check the 'I'm a subcontractor" will open a side panel with added fields.
# Checklist:
